more error tracking docs (specifically rules) (#11924)
PostHog/posthog.com: contents/docs/error-tracking/alerts.mdx

![Slack notification](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_04_08_at_14_02_41_2x_2410295066.png)

### Alert filtering

> You can filter an alert based on the properties of an issue.
> You might find this useful to notify the Slack channel of a team when they have been auto assigned an issue using [auto assignment rules](/docs/error-tracking/managing-issues#auto-assignment-rules).

Reviewer: ```suggestion
You can filter an alert based on the properties of an issue. This is useful for notifying a specific team when they have been auto assigned an issue using [auto assignment rules](/docs/error-tracking/managing-issues#auto-assignment-rules).
```
